### PR TITLE
Added javascript questions

### PR DESCRIPTION
**Pull request by vrashankshetty**
## Summary of changes

Added questions to javascript quiz

## Checklist

- [x] I have read and followed the [contribution guidelines](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CONTRIBUTING.md).
- [x] I have read through the [Code of Conduct](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CODE_OF_CONDUCT.md) and agree to abide by the rules.
- [x] This PR is for one of the available issues and is not a PR for an issue already assigned to someone else.
- [x] My PR title has a short descriptive name so the maintainers can get an idea of what the PR is about.
- [x] I have provided a summary of my changes.

closes #XXXXX

### CODE REVIEW

File: src/data/javascript-quiz.ts [hunk 1]
```diff
@@ -1994,6 +1994,26 @@ const javascriptQuiz = [
...
+    Distractor1: "To create web animations and transitions.",
+    Distractor2: "To manage browser cookies and local storage.",
+    Distractor3:"To enhance the rendering of web pages.",
+    Explanation:
+      "Web Workers are a powerful feature for improving the responsiveness of web applications by offloading CPU-intensive tasks to separate threads. Understanding their use is essential for building efficient web applications",
+    Link: "https://www.freecodecamp.org/news/how-webworkers-work-in-javascript-with-example/"
+  },
+  {
+    Question:"What is 'event delegation' in JavaScript, and why is it useful?",
+    Answer: "Event delegation is a technique for handling events on multiple child elements by attaching a single event listener to a common ancestor.",
```
Comment (jdwilkin4) [inline src/data/javascript-quiz.ts:2010]: short answers are better here

```suggestion
    Answer: "Event delegation is a technique for handling events on multiple child elements",
```

File: src/data/javascript-quiz.ts [hunk 2]
```diff
@@ -1994,6 +1994,26 @@ const javascriptQuiz = [
...
+    Answer: "To perform time-consuming tasks in the background without blocking the main thread.",
+    Distractor1: "To create web animations and transitions.",
+    Distractor2: "To manage browser cookies and local storage.",
+    Distractor3:"To enhance the rendering of web pages.",
+    Explanation:
+      "Web Workers are a powerful feature for improving the responsiveness of web applications by offloading CPU-intensive tasks to separate threads. Understanding their use is essential for building efficient web applications",
+    Link: "https://www.freecodecamp.org/news/how-webworkers-work-in-javascript-with-example/"
+  },
+  {
+    Question:"What is 'event delegation' in JavaScript, and why is it useful?",
```
Comment (jdwilkin4) [inline src/data/javascript-quiz.ts:2009]:
```suggestion
    Question:"What is 'event delegation' in JavaScript?",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by vrashankshetty** [comment]
Made the changes as per requested,
Thank you

**Review by jdwilkin4 — APPROVED** [review]
Congrats on another contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**